settings.gradle.kts

```diff
@@ -30,6 +30,7 @@ dependencyResolutionManagement {
   }
 }
 
+// include(":examples")
```

Comment:
Was it intentional?

Reply:
Sorry, no, because to run Storytale in the examples module, it is necessary to `include(":examples")`. I forgot to remove it